assets/src/edit-story/components/colorPicker/gradientLine.js

```diff
@@ -107,7 +107,10 @@ function GradientLine({
       {stops.map(({ position }, index) => (
         <GradientStop
           ref={(ref) => (stopRefs[index].current = ref)}
-          key={index}
+          key={
+            /* eslint-disable-next-line react/no-array-index-key */
+            index
+          }
```

Comment:
Here, `position` is unique - guaranteed by the reducer.

Reply:
It was `position` originally but it breaks the logic and tests. So I assumed that perhaps the position is changing in a way that's not expected. E.g. when one stop gets deleted?

Reply:
In that case, this is fine. Not sure why it doesn't work though, but fine with me. No need to spend more time looking into this.